health/HealthService.go

```diff
@@ -0,0 +1,33 @@
+package health
```

Comment:
Please use snake_case for filenames. It's a common practice if not the official convention.